semgrep/semgrep/cli.py

```diff
@@ -190,6 +192,18 @@ def cli() -> None:
         ),
     )
 
+    config.add_argument(
+        "--max-target-size",
+        type=parse_size,
```

Comment:
I was confused by this---is parse_size a class? My understanding was that the argument is casted to this type

Reply:
yeah I'd expect `type=int` here.

Reply:
oh nm looks like it's in semgrep.size

Reply:
yeah it has to be a function that takes a string. I might use the `datasize` package Nathan suggests in another thread; I saw an older one mentioned in stackoverflow answers and it seemed overkill.